cargo-crev/src/repo.rs

```diff
@@ -152,6 +152,7 @@ impl Repo {
         let version_str = version.map(ToString::to_string);
         let dependency_request =
             Dependency::parse_no_deprecated(name, version_str.as_deref(), source.source_id())?;
+        let _lock = self.config.acquire_package_cache_lock()?;
```

Comment:
That's it? One place? Hmmm.... OK...

Reply:
If there are more, it's only a matter of adding it where panic backtrace points to.